aws/session/shared_config_test.go

```diff
@@ -507,6 +523,15 @@ func TestLoadSharedConfigFromFile(t *testing.T) {
 				S3UseARNRegion: true,
 			},
 		},
+		{
+			Profile: "sso-session-success",
```

Comment:
**Suggestion**: Would be good to add some of the validation cases here as well, not just the happy path.

Reply:
^agreed